docs/eloquent-models/schema-builder.txt

```diff
@@ -0,0 +1,393 @@
+field:
+
+.. io-code-block::
+
+   .. input:: /includes/schema-builder/planets_migration.php
+      :language: php
+      :dedent:
+      :start-after: begin multi index helpers
+      :end-before: end multi index helpers
+
+   .. output::
+      :language: json
+      :visible: false
+
+      [
+        { v: 2, key: { _id: 1 }, name: '_id_' },
+        {
+          v: 2,
+          key: { last_visible_dt: 1 },
+          name: 'last_visible_dt_1',
+          unique: true,
+          sparse: true,
+          expireAfterSeconds: 3600
+        }
+      ]
+
+To learn more about these indexes, see :manual:`Index Properties </core/indexes/index-properties/>`
+in the {+server-docs-name+}.
+
+Create a Geospatial Index
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In MongoDB, geospatial indexes let you query geospatial coordinate data for
+inclusion, intersection, and proximity.
+
+To create geospatial indexes, call the ``create()`` method on the ``Schema`` facade
+in your migration file. Pass ``create()`` the collection name and a callback
+method with a ``MongoDB\Laravel\Schema\Blueprint`` parameter. Specify the
+geospatial index creation details on the ``Blueprint`` instance.
+
+The following example migration creates a ``2d`` and ``2dsphere`` geospatial
+index on the ``spaceports`` collection. Click the :guilabel:`VIEW OUTPUT`
+button to see the indexes created by running the migration, including the
+default index on the ``_id`` field:
+
+.. io-code-block::
+   .. input:: /includes/schema-builder/spaceports_migration.php
+      :language: php
+      :dedent:
+      :start-after: begin create geospatial index
+      :end-before: end create geospatial index
+
+   .. output::
+      :language: json
+      :visible: false
+
+      [
+        { v: 2, key: { _id: 1 }, name: '_id_' },
+        {
+          v: 2,
+          key: { launchpad_location: '2dsphere' },
+          name: 'launchpad_location_2dsphere',
+          '2dsphereIndexVersion': 3
+        },
+        { v: 2, key: { runway_location: '2d' }, name: 'runway_location_2d' }
+      ]
+
+
+To learn more about geospatial indexes, see
+:manual:`Geospatial Indexes </core/indexes/index-types/index-geospatial/>` in
+the {+server-docs-name+}.
+
+Drop an Index
+~~~~~~~~~~~~~
+
+To drop indexes from a collection, call the ``table()`` method on the
+``Schema`` facade in your migration file. Pass it the table name and a
+callback method with a ``MongoDB\Laravel\Schema\Blueprint`` parameter.
+Call the ``dropIndex()`` method with the index name on the ``Blueprint``
+instance.
+
+.. note::
+
+   If you drop a collection, MongoDB automatically drops all the indexes
+   associated with it.
+
+The following example migration drops an index called ``unique_mission_id_idx``
+from the ``flights`` collection:
+
+.. literalinclude:: /includes/schema-builder/flights_migration.php
+   :language: php
+   :dedent:
+   :start-after: begin drop index
+   :end-before: end drop index
+
+
```

Comment:
It would be interesting to add a section on search index creation. Waiting for a helper to be added: PHPORM-165

Reply:
I created https://jira.mongodb.org/browse/DOCSP-37888 to track this.

I think you might be able to use the "Docs Changes Needed" checkbox in JIRA to generate tickets related to features. This workflow could be more convenient than communicating through PR comments or Slack.